bedrock/firefox/templates/firefox/welcome/page7.html

```diff
@@ -0,0 +1,118 @@
+{# This Source Code Form is subject to the terms of the Mozilla Public
+# License, v. 2.0. If a copy of the MPL was not distributed with this
+# file, You can obtain one at http://mozilla.org/MPL/2.0/. -#}
+
+{% from "macros.html" import google_play_button, send_to_device with context %}
+{% from "macros-protocol.html" import hero with context %}
+
+{% add_lang_files "firefox/welcome/page7" %}
+
+{% extends "firefox/welcome/base.html" %}
+
+{# L10n: HTML page title #}
+{% block page_title %}{{ _('Make it harder for Facebook to track you') }}{% endblock %}
+
+{% block body_class %}{{ super() }} welcome-page7{% endblock %}
+
+{% set show_send_to_device = LANG in settings.SEND_TO_DEVICE_LOCALES %}
+{% set android_url = lockwise_adjust_url('android', 'welcome-7') %}
+{% set ios_url = lockwise_adjust_url('ios', 'welcome-7') %}
+
+{% set _entrypoint = 'mozilla.org-firefox-welcome-7' %}
+{% set _utm_campaign = 'welcome-7-facebook-container' %}
+
+{% block content_intro %}
+{% call hero(
+    title=_('It’s okay to like Facebook'),
+    desc=_('If you still kinda like Facebook but don’t trust them, then try the Facebook Container extension by Firefox and make it harder for them to track you around the web.'),
+    class='mzp-t-firefox mzp-t-dark',
+    include_cta=True,
+    heading_level=1
+  ) %}
+
+  <p class="primary-cta">
+    <a href="https://addons.mozilla.org/firefox/addon/facebook-container/?src=external-www.mozilla.org-whatsnew74&utm_source=www.mozilla.org-whatsnew74&utm_medium=referral&utm_campaign=feb2020" class="mzp-c-button mzp-t-product" id="facebook-container-button" rel="external">{{ _('Get Facebook Container') }}</a>
```

Comment:
These URL params still mention whatsnew74 and need to be updated for this page, here and on the other links. @hoosteeno or @sghoseWI might have input on what the preferred values should be.